pom.xml

```diff
@@ -850,6 +838,19 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>java17-modules</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>java17-modules</name>
```

Comment:
Out of curiousity, why note have an activation that checks for the current JDK?
Instead of `property`, you should be able to use `jdk` and test against `[17,)` (at least 17)

Reply:
Profiles can always be included or excluded by using the `-P` flag in the maven command.

Reply:
I wasn't aware this was possible. I also changed it for the hibernate-6 module, so it's always run when at least java 11. Maven is also capable of ordering the modules based on the dependencies, so for coverage, we only need to add the `coverage-report` module.